deploy/cluster_role_binding_che.yaml

```diff
@@ -0,0 +1,22 @@
+#
+#  Copyright (c) 2012-2019 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: che-operator-che
```

Comment:
Is it cluster_role for Che Server? 
Maybe it's better to show it in its name...

Also, is OLM going to create it for Che Server? Is it the right thing to do? Maybe Che Operator should create it only when Che Server needs it?

Reply:
yes, it is for `che` sa. But operator, to be able to grant these permissions, has to have it on it's own. So I'm creating the ClusterRole now. Do you see anny better approach here?

Reply:
> yes, it is for che sa. But operator, to be able to grant these permissions, has to have it on it's own. So I'm creating the ClusterRole now. Do you see anny better approach here?

it's fine for me, I just found that `che-operator-che` RB name tells nothing. Maybe it would be better to name it as `che-operator-cluster-role`.

Reply:
Ok